### PR TITLE
Bilous s

### PR DESCRIPTION
**Pull request by BelousSofiya**

### CODE REVIEW

File: BilousS/HW10/Practical task(outside)/Ball Super Ball.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Ball(object):
```
Comment (kolyasalubov) [inline BilousS/HW10/Practical task(outside)/Ball Super Ball.py:1]: good job

File: BilousS/HW10/Practical task(outside)/Basic subclasses - Adam and Eve.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Human:
```
Comment (kolyasalubov) [inline BilousS/HW10/Practical task(outside)/Basic subclasses - Adam and Eve.py:1]: good job

File: BilousS/HW10/Practical task(outside)/Classy Classes.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class Person:
```
Comment (kolyasalubov) [inline BilousS/HW10/Practical task(outside)/Classy Classes.py:1]: good implementation

File: BilousS/HW10/Practical task(outside)/Building Spheres.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+from math import pi
```
Comment (kolyasalubov) [inline BilousS/HW10/Practical task(outside)/Building Spheres.py:1]: good implementation

File: BilousS/HW10/Practical task(outside)/Color Ghost.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+from random import choice
```
Comment (kolyasalubov) [inline BilousS/HW10/Practical task(outside)/Color Ghost.py:1]: good implementation

File: BilousS/HW10/Practical task(outside)/Python's Dynamic Classes #1.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+def class_name_changer(cls,new_name):
```
Comment (kolyasalubov) [inline BilousS/HW10/Practical task(outside)/Python's Dynamic Classes #1.py:1]: good job

File: BilousS/HW10/task1.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
+class Poligon:
```
Comment (kolyasalubov) [inline BilousS/HW10/task1.py:1]: good job

File: BilousS/HW10/task2.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
+class Human:
```
Comment (kolyasalubov) [inline BilousS/HW10/task2.py:1]: good implementation

File: BilousS/HW10/task3.py [hunk 9]
```diff
@@ -0,0 +1,69 @@
+class Employee:
```
Comment (kolyasalubov) [inline BilousS/HW10/task3.py:1]: good job

File: BilousS/HW11/task1_from_presentation.py [hunk 10]
```diff
@@ -0,0 +1,8 @@
+number = input('Write the integer: ')
```
Comment (kolyasalubov) [inline BilousS/HW11/task1_from_presentation.py:1]: good job

File: BilousS/HW11/task2_from_presenttion.py [hunk 11]
```diff
@@ -0,0 +1,14 @@
+while True:
```
Comment (kolyasalubov) [inline BilousS/HW11/task2_from_presenttion.py:1]: good job

File: BilousS/HW11/task3_from_presentation.py [hunk 12]
```diff
@@ -0,0 +1,17 @@
+def check(age):
```
Comment (kolyasalubov) [inline BilousS/HW11/task3_from_presentation.py:1]: good job

File: BilousS/HW11/task4_from_presentation.py [hunk 13]
```diff
@@ -0,0 +1,23 @@
+days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
```
Comment (kolyasalubov) [inline BilousS/HW11/task4_from_presentation.py:1]: good job

File: BilousS/HW10/Practical task(outside)/Ball Super Ball.py (hunk 1 above)
Comment (BelousSofiya) [reply BilousS/HW10/Practical task(outside)/Ball Super Ball.py:1]: Thanks

File: BilousS/HW10/Practical task(outside)/Classy Classes.py (hunk 3 above)
Comment (BelousSofiya) [reply BilousS/HW10/Practical task(outside)/Classy Classes.py:1]: Thanks

File: BilousS/HW10/Practical task(outside)/Basic subclasses - Adam and Eve.py (hunk 2 above)
Comment (BelousSofiya) [reply BilousS/HW10/Practical task(outside)/Basic subclasses - Adam and Eve.py:1]: Thanks

**Pull request merged**